light_curves/light_curve_generator.md

```diff
@@ -74,7 +74,7 @@ This cell will install them if needed:
 
 ```{code-cell} ipython3
 # Uncomment the next line to install dependencies if needed.
-# !pip install -r requirements_light_curve_generator.txt
+!pip install -r requirements_light_curve_generator.txt
```

Comment:
This doesn't make CI happy :)

The other failures have already been fixed on the default branch.

Reply:
Is there anything we can do about this because I need it to run the notebooks, and you need it not to be there for CI?  Somehow remembering to comment it out isn't sticking in my brain.  Can we wrap this in an if statement or something?  It is not worth a lot of effort, but if there is a workaround that keeps you from having to remind me every single time, that might be worth it.  And sorry I can't seem to remember this.